spring-boot-starter/example-invoice/src/main/resources/application.yaml

```
@@ -0,0 +1,38 @@
+spring.datasource:
+  url: jdbc:h2:./camunda-h2-default/process-engine;TRACE_LEVEL_FILE=0;DB_CLOSE_ON_EXIT=FALSE
+#  url: jdbc:postgresql://localhost:5432/process-engine
+#  url: jdbc:mysql://localhost:3306/process-engine
+#  url: jdbc:mariadb://localhost:3306/process-engine
+#  url: jdbc:oracle:thin:@localhost:1521:ORCL
+#  url: jdbc:db2://localhost:50000:engine
+#  url: jdbc:sqlserver://localhost:1433;databaseName=processEngine
+  username: camunda
+  password: camunda
+  driver-class-name: org.h2.Driver
+#  driver-class-name: org.postgresql.Driver
+#  driver-class-name: com.mysql.jdbc.Driver
+#  driver-class-name: org.mariadb.jdbc.Driver
+#  driver-class-name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+#  driver-class-name: oracle.jdbc.OracleDriver
+#  driver-class-name: com.ibm.db2.jcc.DB2Driver
+
+camunda:
+  bpm:
+    filter:
+      create: All
+    job-execution:
+      core-pool-size: 10
+    authorization.enabled: true
+    admin-user:
+      id: demo
+      password: demo
```

As discussed remove default filter and default `admin-user`, they will be created by the invoice app.
```suggestion
    job-execution:
      core-pool-size: 10
    authorization.enabled: true
```